### DISCUSSION TITLE
If you use localization for the extension name, the translation key is displayed, not the translation itself

### DISCUSSION THREAD

**Discussion by AlexHodzitsky** [Shopify/function-examples#374] [Feedback: Discounts APIs]
Hi there.

When using translations for the extension name in the config file, in the Shopify admin panel the translation key is displayed instead of the translation itself. This happens when using extensions related to discounts, in my case Product Discount Functions API.

I took the App sample for the test from this Shopify repository: https://github.com/Shopify/function-examples/tree/main/sample-apps/discounts

**Expecting name:** Product Discount (JavaScript)
**Actual name:** t:name

Extension config and locale file:
![image](https://github.com/Shopify/function-examples/assets/63909246/71e5040b-b0d0-4e63-90de-a1707ccf2604)

Output in Shopify > Discounts:
![image](https://github.com/Shopify/function-examples/assets/63909246/0343adfc-6e60-4443-abe0-044edd627ac7)
![image](https://github.com/Shopify/function-examples/assets/63909246/c72ca843-82e1-48c5-bed0-1cb523069721)

**Comment by nickwesselman** [comment]
Hi @AlexHodzitsky -- Thanks for reporting. Fix for this is in the works.

**Comment by nickwesselman** [comment]
Moving to discounts as this is a discount-specific issue.
